### ISSUE TITLE
FAILED open or readN/model.data-00000of-0001

### ISSUE THREAD

**Issue by ygbingo** [IBM/MAX-Image-Resolution-Enhancer#52]
When I build docker image locallly, I got this wrong:
```shell
 > [7/7] RUN sha512sum -c sha512sums.txt:
#11 0.460 sha512sum: 'assets/SRGAN/model.data-00000-of-00001'$'\r': No such file or directory
: FAILED open or readN/model.data-00000-of-00001
#11 0.460 sha512sum: 'assets/SRGAN/model.index'$'\r': No such file or directory
: FAILED open or readN/model.index
#11 0.460 sha512sum: 'assets/SRGAN/model.meta'$'\r': No such file or directory
#11 0.460 sha512sum: WARNING: 3 listed files could not be read
: FAILED open or readN/model.meta
------
```

How could I got these files named "model.data*, model.index, model.meta" in assets/SRGAN? I just want run up at first.

**Comment by AlexKalopsia** [comment]
Running on the same issue when trying to build the docker image

```
sha512sum: 'assets/SRGAN/model.data-00000-of-00001'$'\r': No such file or directory
sha512sum: 'assets/SRGAN/model.index'$'\r': No such file or directory
sha512sum: 'assets/SRGAN/model.meta'$'\r': No such file or directory
sha512sum: WARNING: 3 listed files could not be read
```

**Comment by soladragon** [comment]
Try updating the dockerfile ARG model_bucket link with:

ARG model_bucket=https://codait-cos-max.s3.us.cloud-object-storage.appdomain.cloud/max-image-resolution-enhancer/1.0.0

**Comment by kmh4321** [comment]
Thank you for your patience, this is now fixed in: 
https://github.com/IBM/MAX-Image-Resolution-Enhancer/pull/55

**Comment by tpmsh** [comment]
This issue has again started appearing. On going to the url 
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Resource>/codait-cos-max/max-image-resolution-enhancer//1.0.0/assets.tar.gz</Resource>
<RequestId>fec17957-c656-4bc0-98ec-d16487b0feea</RequestId>
<httpStatusCode>404</httpStatusCode>
</Error>

For
 => CACHED [5/6] COPY . .                                                                                          0.0s
 => ERROR [6/6] RUN sha512sum -c sha512sums.txt                                                                    0.3s
------
 > [6/6] RUN sha512sum -c sha512sums.txt:
#0 0.274 sha512sum: 'assets/SRGAN/model.data-00000-of-00001'$'\r': No such file or directory
#0 0.274 sha512sum: 'assets/SRGAN/model.index'$'\r': No such file or directory
#0 0.274 sha512sum: 'assets/SRGAN/model.meta'$'\r': No such file or directory
: FAILED open or read/model.data-00000-of-00001
: FAILED open or read/model.index
: FAILED open or read/model.meta
#0 0.275 sha512sum: WARNING: 3 listed files could not be read

So basically it is not able to download the assets from the S3 Bucket.
